site/utils/index.ts

```diff
@@ -281,3 +282,35 @@ export const getStopTypesByVehicleMode = (vehicleMode: VehicleMode | Modes) => {
             return { stopTypes: "undefined" };
     }
 };
+
+const getRandomInt = (max: number): number => {
+    let randomNumber: number;
+    do {
+        randomNumber = randomBytes(1)[0];
+    } while (randomNumber >= 256 - (256 % max));
+    return randomNumber % max;
+};
+
+export const generatePassword = (length: number): string => {
```

Comment:
It's worth having a function comment explaining why the code is in the func is included, i.e. ensuring the password has at least one type of each character, filling the length and finally shuffling the chars